pom.xml

```diff
@@ -26,7 +26,8 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cdap.version>6.1.0-SNAPSHOT</cdap.version>
+    <cdap.version>6.1.2</cdap.version>
```

Comment:
should be 6.1.1

Reply:
We tried with 6.1.1 and there was no method getRawProperties(), is this added on 6.1.2?

Reply:
I see

Reply:
I see